LetsDinnerV2 MessagesExtension/Manager/ActionSheetManager.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  ActionSheetManager.swift
+//  LetsDinnerV2 MessagesExtension
+//
+//  Created by Alex Cheung on 1/7/2020.
+//  Copyright © 2020 Eric Ordonneau. All rights reserved.
+//
+
+import UIKit
+
+class ActionSheetManager {
```

Comment:
No you can't do that, this is all but generic. You just put action sheets in a single class? And then each method is specific to just one place in the app? What is the point? You remove all the privacy. Then why not put all the labels in a label manager class, the button in a button manager class? This stuff means i can call presentSaveActionSheetInRegistrationVC in any view controller. This is bad

Reply:
no you can only call it from registrationVC

Reply:
Generalized